bitnami/contour/README.md

```diff
@@ -153,24 +173,66 @@ The following tables lists the configurable parameters of the contour chart and
+| `defaultBackend.image.repository`          | Default backend image name                                                               | `bitnami/nginx`                                         |
+| `defaultBackend.image.tag`                 | Default backend image tag                                                                | `{TAG_NAME}`                                            |
+| `defaultBackend.image.pullPolicy`          | Image pull policy                                                                        | `IfNotPresent`                                          |
+| `defaultBackend.image.pullSecrets`         | Specify docker-registry secret names as an array                                         | `[]` (does not add image pull secrets to deployed pods) |
+| `defaultBackend.extraArgs`                 | Additional command line arguments to pass to NGINX container                             | `{}`                                                    |
+| `defaultBackend.containerPort`             | HTTP container port number                                                               | `8080`                                                  |
+| `defaultBackend.replicaCount`              | Desired number of default backend pods                                                   | `1`                                                     |
+| `defaultBackend.podSecurityContext`        | Default backend pods' Security Context                                                   | Check `values.yaml` file                                |
+| `defaultBackend.containerSecurityContext`  | Default backend containers' Security Context                                             | Check `values.yaml` file                                |
+| `defaultBackend.resources.limits`          | The resources limits for the Default backend container                                   | `{}`                                                    |
+| `defaultBackend.resources.requests`        | The requested resources for the Default backend container                                | `{}`                                                    |
+| `defaultBackend.leavinessProbe`            | Leaviness probe configuration for Default backend                                        | Check `values.yaml` file                                |
+| `defaultBackend.readinessProbe`            | Readiness probe configuration for Default backend                                        | Check `values.yaml` file                                |
+| `defaultBackend.customLivenessProbe`       | Override default liveness probe                                                          | `nil`                                                   |
+| `defaultBackend.customReadinessProbe`      | Override default readiness probe                                                         | `nil`                                                   |
```

Comment:
This 2 parameters seem to be missing from `values.yaml`.

Reply:
Done